build/relsignDSCModules.sh

```diff
@@ -0,0 +1,309 @@
+    output=$(curl --silent --write-out %{http_code} --request "GET" \
+                  --header "x-ms-date    : $timestamp" \
+                  --header "x-ms-version : $StorageVersion" \
+                  "$url" )
+    
+    code=${output: (-3)}	
+	if [ "$code" != 200 ]; then
+	    echo "read_blob(): error retrieving $remotefile, server returned an http error code $code"
+        return 1
+    fi
+
+    output=${output: :(-3)}
+    echo -n "$output" > "$localfile"
+   
+    return 0
+}
+
+# write_blob <SASURL> <FILE>
+write_blob() {
+
+    [ ! -z "$1" ] || error "write_blob(): expected remote URI" 1
+    [ ! -z "$2" ] || error "write_blob(): expected filename to upload" 1
+    [ -f "$2" ] || error "write_blob(): file $2 does not exist" 1
+
+    url="$1"
+    localfile="$2"
+    remotefile=$(basename "$localfile")
+
+    container=$( echo "$1" | awk -F \?  '{container=index($0,"?"); print $1}' )
+	echo "container is $container"
+    sas=$( echo "$url" | awk -F \? '{container=index($0,"?"); print $2}' )
+    url="${container}/${remotefile}?${sas}"
+
+    timestamp=$(date +%Y-%m-%dT%H:%M:%SZ)
+    output=$(curl --silent --write-out %{http_code} --request "PUT" \
+                  --header "x-ms-date      : $timestamp" \
+                  --header "x-ms-blob-type : BlockBlob" \
+                  --header "x-ms-version   : $StorageVersion" \
+                  --data-binary "@${localfile}" "$url" )
+    
+    code=${output: (-3)}
+	if [ "$code" != 201 ]; then
+        echo "write_blob(): error writing $remotefile, server returned an http error code $code"
+		return 1
+    fi
+
+    return 0
+}
+
+# delete <URI> <REMOTE_FILE>
+delete_blob() {
+
+    [ ! -z "$1" ] || error "delete_blob(): expected remote URI" 1
+    [ ! -z "$2" ] || error "delete_blob(): expected remote filename" 1
+   
+    url="$1"
+    remotefile="$2"
+    
+    container=$( echo "$1" | awk -F \?  '{container=index($0,"?"); print $1}' )
+	echo "container is $container"
+    sas=$( echo "$url" | awk -F \? '{container=index($0,"?"); print $2}' )
+    url="${container}/${remotefile}?${sas}"
+    echo "url is $url"
+    timestamp=$(date +%Y-%m-%dT%H:%M:%SZ)
+    output=$(curl --silent --write-out %{http_code} --request "DELETE" \
+                  --header "x-ms-date    : $timestamp" \
+                  --header "x-ms-version : $StorageVersion" \
+                  "$url" )
+    
+    code=${output: (-3)}
+    if [[ "$code" != 202 ]]  && [[ "$code" != 404 ]]; then
+        echo "delete_blob(): error deleting $remotefile, server returned error code $code"
+		return 1
+    fi    
+    return 0
+}
+# SAS URLs
+SAS_SOURCE_URI=$1
+echo " SAS source container URL is : $1"
+SAS_DEST_URI=-$2
+echo " SAS destination container URL is : $2"
+# Build info
+BUILD_VERS=$3
+BUILD_BASEDIR=$4
+
+if [ "$BUILD_BASEDIR" = "" ]; then
+
+    echo "No build share path is specified. Default build share is mounted at: /mnt/omsdata"
+
+    BUILD_BASEDIR=/mnt/omsdata/OMSData/Builds/omsagent/develop
```

Comment:
This is just a default. The default really is not needed, but just sake of convenience. It can be used for OSTC builds as well as long as you pass parameter $4.